lib/server/command_processor.js

```diff
@@ -77,6 +82,18 @@ class CommandProcessor extends Duplex {
         Promise.resolve().then(() => this._read_internal());
     }
 
+    /**
+     * @private
+     */
+    _isWhitelisted(ip) {
+        if(this._whitelistEmpty){
```

Comment:
can be simplified:

`return this._whitelistEmpty || this._putWhitelist.indexOf(ip) != -1`